steps/appeal_steps.js

```diff
@@ -0,0 +1,35 @@
+import { expect } from '@playwright/test'
+
+export class AppealSteps {
+  constructor(page) {
+    this.page = page
+  }
+
+// TODO: test this method
```

Comment:
I think either we need to resolve this TODO before merging, or we need to ticket it up and have this comment reference the ticket. Otherwise this comment just says "hey, this method might not work right and we're maybe going to do something at some point about it but maybe not"